### PR TITLE
nos3#100 - Generic Radio

### PR DESCRIPTION
**Pull request by jlucas9**
Ready to review the generic radio baseline!

Can be merged once the following submodule merges are completed:
- [x] https://github.com/nasa-itc/generic_radio/pull/1
- [x] https://github.com/nasa-itc/gsw-cosmos/pull/10

### CODE REVIEW

File: fsw/nos3_defs/tables/to_config.c [hunk 1]
```diff
@@ -141,8 +142,8 @@ TO_ConfigTable_t to_ConfigTable =
        /* 40 - 49 */
        {GENERIC_CSS_HK_TLM_MID,       {0,0},  32,  0xffff,     TO_GROUP_APP | TO_MGROUP_ONE, 0,1},
        {GENERIC_CSS_DEVICE_TLM_MID,   {0,0},  32,  0xffff,     TO_GROUP_APP | TO_MGROUP_ONE, 0,1},
-       {TO_UNUSED_ENTRY,              {0,0},  0,   0x0000,     TO_GROUP_NONE,            0,0},
-       {TO_UNUSED_ENTRY,              {0,0},  0,   0x0000,     TO_GROUP_NONE,            0,0},
+       {GENERIC_RADIO_HK_TLM_MID,     {0,0},  32,  0xffff,     TO_GROUP_APP | TO_MGROUP_ONE, 0,1},
+       {GENERIC_RADIO_DEVICE_TLM_MID, {0,0},  32,  0xffff,     TO_GROUP_APP | TO_MGROUP_ONE, 0,1},
```
Comment (msuder) [inline fsw/nos3_defs/tables/to_config.c:146]: Delete this line.  There is no GENERIC_RADIO_DEVICE_TLM_MID packet.

File: fsw/nos3_defs/tables/to_lab_sub.c [hunk 2]
```diff
@@ -89,8 +90,10 @@ TO_LAB_Subs_t TO_LAB_Subs =
         {CFE_SB_MSGID_WRAP_VALUE(GENERIC_EPS_HK_TLM_MID),       {0,0},  32},
         {CFE_SB_MSGID_WRAP_VALUE(GENERIC_FSS_HK_TLM_MID),       {0,0},  32},
         {CFE_SB_MSGID_WRAP_VALUE(GENERIC_FSS_DEVICE_TLM_MID),   {0,0},  32},
+        {CFE_SB_MSGID_WRAP_VALUE(GENERIC_RADIO_HK_TLM_MID),     {0,0},  32},
+        {CFE_SB_MSGID_WRAP_VALUE(GENERIC_RADIO_DEVICE_TLM_MID), {0,0},  32},
```
Comment (msuder) [inline fsw/nos3_defs/tables/to_lab_sub.c:94]: Delete this line.  See above.

File: fsw/nos3_defs/targets.cmake [hunk 3]
```diff
@@ -117,7 +117,8 @@ SET(APPLICATION_LIST
         generic_eps
         generic_fss
         generic_reaction_wheel
-	generic_torquer
+        generic_radio
+	    generic_torquer
```
Comment (msuder) [inline fsw/nos3_defs/targets.cmake:121]: Fix spacing.  Tabs should not be used; indentation should be 4 spaces.

**Review by msuder — CHANGES_REQUESTED** [review]

File: fsw/nos3_defs/tables/to_config.c (hunk 1 above)
Comment (jlucas9) [reply fsw/nos3_defs/tables/to_config.c]: Resolved in commit 38dd658

File: fsw/nos3_defs/tables/to_lab_sub.c (hunk 2 above)
Comment (jlucas9) [reply fsw/nos3_defs/tables/to_lab_sub.c]: Resolved in commit 38dd658

File: fsw/nos3_defs/targets.cmake (hunk 3 above)
Comment (jlucas9) [reply fsw/nos3_defs/targets.cmake]: Resolved in commit 38dd658

**Pull request merged**